src/pages/settings/bank-accounts/create/Create.tsx

```diff
@@ -54,17 +57,21 @@ export function Create() {
     setBankAccount((prevState) => ({ ...prevState, [property]: value }));
   };
 
+  const invalidateCache = () => {
+    queryClient.invalidateQueries(route('/api/v1/bank_integrations'));
+  };
+
   const handleCancel = () => {
     if (!isFormBusy) {
-      navigate(route('/settings/bank_accounts'));
+      navigate(route('/settings/bank_accounts/'));
```

Comment:
As previously mentioned, no need to use `route` when there are no query or URL parameters. It does nothing.

Reply:
> As previously mentioned, no need to use `route` when there are no query or URL parameters. It does nothing.

Removed